### PR TITLE
replace makefiles with bash script

### PR DESCRIPTION
**Pull request by tinmarr**

### CODE REVIEW

File: b [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    fi
+}
+
+delete() {
+    case $1 in
+        bin|b)
+            echo "Deleting binary files..."
+            find . -name "*.pyd" -type f -delete
+            find . -name "*.so" -type f -delete
+            ;;
```
Comment (mrElnekave) [inline b:42]: is it ; or ;;?

File: b [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        c)
+            echo "Deleting c files..."
+            find . -name "*.cpp" -not -name "cdraw.cpp" -type f -delete
+            find . -name "*.c" -type f -delete
+            ;;
+        *)
+            echo "Deleting build directory..."
+            rm -rf build
+            delete bin
+            delete c
```
Comment (mrElnekave) [inline b:52]: do you not need ;; here?

File: b (hunk 1 above)
Comment (tinmarr) [reply b:42]: ;;

File: b (hunk 2 above)
Comment (mrElnekave) [inline b:52]:
```suggestion
            delete c
            ;;
```

File: b [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+            ;;
```
Comment (mrElnekave) [inline b:53]: Martin is dis goog?

**Review by tsedan — APPROVED** [review]
goog

**Pull request merged**